tests/test_minhash.py

```diff
@@ -2650,3 +2650,150 @@ def test_containment(track_abundance):
 
     assert mh1.contained_by(mh2) == 1/4
     assert mh2.contained_by(mh1) == 1/2
+
+
+def test_containment_ANI():
+    f1 = utils.get_test_data('2.fa.sig')
+    f2 = utils.get_test_data('2+63.fa.sig')
+    f3 = utils.get_test_data('47+63.fa.sig')
+    mh1 = sourmash.load_one_signature(f1, ksize=31).minhash
+    mh2 = sourmash.load_one_signature(f2, ksize=31).minhash
+    mh3 = sourmash.load_one_signature(f3, ksize=31).minhash
+
+    print("\nmh1 contained by mh2", mh1.containment_ani(mh2, return_ci =True))
+    print("mh2 contained by mh1",mh2.containment_ani(mh1, return_ci=True))
+    print("mh1 max containment", mh1.max_containment_ani(mh2, return_ci=True))
+    print("mh2 max containment", mh2.max_containment_ani(mh1, return_ci=True))
+
+    print("\nmh2 contained by mh3", mh2.containment_ani(mh3))
+    print("mh3 contained by mh2",mh3.containment_ani(mh2))
+
+    print("\nmh2 contained by mh3, CI 90%", mh2.containment_ani(mh3, confidence=0.9, return_ci=True))
+    print("mh3 contained by mh2, CI 99%",mh3.containment_ani(mh2, confidence=0.99, return_ci=True))
+
+    assert mh1.containment_ani(mh2,return_ci=True) == (1.0, 1.0, 1.0, 0.0)
+    assert mh2.containment_ani(mh1, return_ci=True) == (0.9658183324254062, 0.9648452889933389, 0.966777042966207, 0.0)
```

Comment:
suggest using `round` here to bring down to ~2-3 significant digits.

Reply:
I assume you mean: `round` all output out of `containment_to_distance` and `jaccard_to_distance`, rather than just this test, right?

Reply:
yep ;).

Reply:
👍🏻!